react/features/base/dialog/components/web/DialogContainer.js

```diff
@@ -20,10 +20,6 @@ class DialogContainer extends AbstractDialogContainer {
      * @returns {ReactElement}
      */
     render() {
-        if (this.props._rawDialog) {
-            return this._renderDialogContent();
-        }
-
```

Comment:
should we get rid of `rawDialog` from store and action signature as well? Noticed `openDialog` is never called with it as true

Reply:
oh I thought I had caught them all. Yes.